src/DeviceInput/InputDevices/deviceTypes.ts

```diff
@@ -10,4 +10,9 @@ export type DeviceInput<T extends DeviceType> =
     T extends DeviceType.Xbox ? XboxInput :
     T extends DeviceType.Switch ? SwitchInput :
     T extends DeviceType.DualSense ? DualSenseInput :
-    never;
\ No newline at end of file
+    never;
+
+    /**
+     * Type to define functions that handle connect/disconnect fo devices
+     */
+export type DeviceStatusChangedCallback = (deviceType: DeviceType, deviceSlot: number) => void;
```

Comment:
I know I suggested this when we talked, but I'm not sure it makes sense to do this if we can't do it to all 3 callbacks. Maybe it would make more sense to add this once the native contact catches up.

Reply:
Removed and will address at a later time